### DISCUSSION TITLE
Snapmod active indication

### DISCUSSION THREAD

**Discussion by doteq** [rodit/SnapMod#170] [Ideas & Feature Suggestions]
I'm using snapmod for sending pictures from gallery, because it takes much better photos. The problem is you can't tell if snapmod works, and if you'll send picture as live snap. When snapchat updates, snapmod is no longer working and because of that I accidentally sent photos from gallery as chat message and it was awkward. So it would be great to have small indication, that you are really sending picture as live snap.

**Comment by rodit** [comment]
For now I'd recommend turning off auto updates, but I may add this in the future.

**Comment by Magna-Graecia** [comment]
Honestly, a lot of these feature requests lately are just completely unnecessary and coming from a point of pure laziness. There's intuitive convenience, and then there's absurdity.  "Can you please add this bloatware to clutter the app some more so I don't have to use my brain at any point whatsoever during my usage of the mod? Thanks!" Between snap save indicators, snap saving overlay buttons, and now SnapMod active indicators, I'll be surprised if soon I'm even able to properly view any snaps without a bunch of crap cluttering the view with how the requests are going lately.

Disable automatic updates for Snapchat and problem solved. Snapmod is always active if you're on a currently supported version of Snapchat, which you should be if you're updating it manually, and you can't expect an indicator for every single setting in Snapmod to let you know if it's currently enabled or disabled. Sometimes you just need to use your brain and remember what settings you have, especially when using a feature in a critical moment. Otherwise don't be lazy and take 2 seconds to check the app if making sure that your sent pic is indeed going to be sent live is so important.

**Reply by Radzi11** [reply]
yes snapmod Indicator or logo when turned on via Snapchat open It would be very useful

**Reply by Magna-Graecia** [reply]
Please explain how, considering we already know that Snapmod is active 24/7 because that's simply how it works. So how would an indicator telling you what you already know be beneficial and not just bloatware? Now if you're talking about an indicator to let you know if individual SnapMod settings are currently enabled/disabled, then sure, but we're going to have an indicator for each individual setting? Really? I don't think so.
